dockerfiles/centos/is/Dockerfile

```diff
@@ -0,0 +1,79 @@
+# ------------------------------------------------------------------------
+#
+# Copyright 2018 WSO2, Inc. (http://wso2.com)
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License
+#
+# ------------------------------------------------------------------------
+# set to latest Centos
+FROM centos:7
+MAINTAINER WSO2 Docker Maintainers "dev@wso2.org”
+
+# set user configurations
+ARG USER=wso2carbon
```

Comment:
We usually use 'wso2' as the user.

Reply:
Hi Kasunbg, 
"wso2carbon" is the standard username that we have been using for a while and "wso2" is the group name.